sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldValue.cs

```diff
@@ -24,7 +24,36 @@ internal FieldValue(FieldValue_internal fieldValue, IReadOnlyList<ReadResult> re
         }
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="FieldValue"/> structure.
+        /// Initializes a new instance of the <see cref="FieldValue"/> structure. Used for mocking purposes.
+        /// </summary>
+#pragma warning disable CA1801 // Review unused parameters
+        internal FieldValue(FieldValueType type, string valueString, DateTimeOffset? valueDate, TimeSpan? valueTime, string valuePhoneNumber, float? valueNumber, long? valueInteger, IReadOnlyList<FieldValue_internal> valueArray, IReadOnlyDictionary<string, FieldValue_internal> valueObject, SelectionMarkState? valueSelectionMark, FieldValueGender? valueGender, string valueCountry, string text)
```

Comment:
I need a method that gives the ability to pass type and text and not necessarily the valueType